erpnext_com/www/docs/user/manual/en/non_profit/index.md

```diff
@@ -6,15 +6,15 @@ People who change the world need the tools to do it! The Non Profit Modules of E
 ## Topics
 
 ##### 1. Membership
-1. [Member](/docs/user/manual/en/non_profit/member)
-1. [Membership Type](/docs/user/manual/en/non_profit/membership_type)
-1. [Membership](/docs/user/manual/en/non_profit/membership)
+1. [Member](/docs/user/manual/en/non_profit/Membership/member)
```

Comment:
these paths do not exists, had removed them long back

Reply:
@prasadarr had sent a PR to fix the paths. Earlier the directories were made and links were changed with the md files not moved in those directories.

This PR moves the md files into those directories and corrects the links accordingly.

Reply:
i removed the directories, makes updating tedious. So not needed